vfox-restServer/src/main/resources/dbScript/baseV0.1.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE IF NOT EXISTS `vfox`.`User` (
   `CreatedAt` DATETIME NOT NULL,
   `UpdatedBy` INT NOT NULL,
   `UpdatedAt` DATETIME NOT NULL,
-  PRIMARY KEY (`Id`),
+  PRIMARY KEY (`Id`,`UserName`),
```

Comment:
Why are we changing user key?

Reply:
Handle Duplicate Records in User table.

Reply:
Why can't we add 'Unique Index' on 'UserName' instead of making it part of Primary key.

Reply:
Indexes are used to retrieve data from the database very fast.

Reply:
Change has been done.